modules/jvm/api/v9/module.yaml

```diff
@@ -0,0 +1,59 @@
+schema_version: 1
+name: jboss.container.java.jvm.api
+version: '9.0'
+description: ^
+  Defines environment variables used for configuring Java JVM, e.g. GC settings,
+  etc.  Modules providing support for configuring JVM settings should use the
+  variables listed within this module to configure the JVM appropriately.
+
+envs:
+- name: JAVA_OPTS
+  description: JVM options passed to the `java` command.
+  example: "-verbose:class"
+- name: JAVA_OPTS_APPEND
+  description: User specified Java options to be appended to generated options in JAVA_OPTS.
+  example: "-Dsome.property=foo"
+- name: JAVA_MAX_MEM_RATIO
+  description: Is used when no `-Xmx` option is given in **JAVA_OPTS**. This is used to calculate a default maximal heap memory based on a containers restriction. If used in a container without any memory constraints for the container then this option has no effect. If there is a memory constraint then `-Xmx` is set to a ratio of the container available memory as set here. The default is `50` which means 50% of the available memory is used as an upper boundary. You can skip this mechanism by setting this value to `0` in which case no `-Xmx` option is added.
+- name: JAVA_INITIAL_MEM_RATIO
+  description: Is used when no `-Xms` option is given in **JAVA_OPTS**. This is used to calculate a default initial heap memory based on the maximum heap memory. If used in a container without any memory constraints for the container then this option has no effect. If there is a memory constraint then `-Xms` is set to a ratio of the `-Xmx` memory as set here. The default is `25` which means 25% of the `-Xmx` is used as the initial heap size. You can skip this mechanism by setting this value to `0` in which case no `-Xms` option is added.
+  example: "25"
```

Comment:
Should we drop those in favor of `-XX:MaxRAMPercentage` and `-XX:InitialRAMPercentage` (and relevant related env var)?